pypeit/core/wavecal/wvutils.py

```diff
@@ -395,12 +400,37 @@ def zerolag_shift_stretch(theta, y1, y2):
     y2_corr = shift_and_stretch(y2, shift, stretch)
     # Zero lag correlation
     corr_zero = np.sum(y1*y2_corr)
-    corr_denom = np.sqrt(np.sum(y1*y1)*np.sum(y2*y2))
+    corr_denom = np.sqrt(np.sum(y1*y1)*np.sum(y2_corr*y2_corr))
     corr_norm = corr_zero/corr_denom
     return -corr_norm
 
-def smooth_ceil_cont(inspec1, smooth, percent_ceil = None, use_raw_arc=False,sigdetect = 10.0, fwhm = 4.0):
-    """ Utility routine to smooth and apply a ceiling to spectra """
+
+def smooth_ceil_cont(inspec1, smooth, percent_ceil = None, use_raw_arc=False, sigdetect = 10.0, fwhm = 4.0,
+                     large_scale_fwhm_fact=20.0, large_scale_sigrej=20.0):
+    """  Utility routine to smooth and apply a ceiling to spectra
+
+    Args:
+        inspec1 (ndarray):
+          Input spectrum, shape = (nspec,)
+        smooth (int):
+          Number of pixels to smooth by
+        percent_ceil (float):
```

Comment:
indicate which ones are `optional`  [most of them]

Reply:
Done.